server/settings/publishers.py

```diff
@@ -1505,6 +1511,11 @@ class PublishersModel(BaseSettingsModel):
         "optional": False,
         "active": True
     },
+    "ValidateAnimationProductTypePublish": {
+        "enabled": True,
```

Comment:
I'd say this is a studio choise, and should be disabled by default because allow multiples to be exported is totally fine behavior... and is 99% of the time a settings config misconfiguration. With this PR a studio can just disable e.g. Alembic project wide and the validator can then be disabled too. I'd disable by default.

```suggestion
        "enabled": False,
```